crates/turbopack-ecmascript-runtime/js/src/dev/runtime/dom/runtime-backend-dom.ts

```diff
@@ -40,6 +40,17 @@ async function loadWebAssembly(
   return instance.exports;
 }
 
+async function loadWebAssemblyModule(
+  _source: SourceInfo,
+  wasmChunkPath: ChunkPath
+): Promise<WebAssembly.Module> {
+  const chunkUrl = `/${getChunkRelativeUrl(wasmChunkPath)}`;
+
+  const req = fetch(chunkUrl);
+
+  return WebAssembly.compileStreaming(req);
```

Comment:
Nit: there's a small perf and dx improvement by awaiting the promise before returning
```suggestion
  return await WebAssembly.compileStreaming(req);
```